test/core/miner/deal_info_manager_test.cpp

```diff
@@ -25,8 +25,10 @@ namespace fc::mining {
 
   template <typename F>
   auto mockSearch(F f) {
-    return testing::Invoke(api::waitCb<boost::optional<MsgWait>>(
-        [f](auto, auto, auto, auto, auto cb) { cb(f()); }));
+    return testing::Invoke([f](auto cb, auto, auto, auto, auto) {
+      cb(f());
+      return outcome::success();
+    });
```

Comment:
```suggestion
    return testing::Invoke([f](auto, auto, auto, auto) {
      return f();
    });
```
maybe can be simplified (with sync MOCK_API)